src/WebJobs.Script/Dispatch/EmptyProcessRegistry.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Seems like the new Dispatch and Rpc folders you created could be combined, so all the "language worker" rpc invoke/channel stuff is under one root directory?